features/call/impl/src/main/kotlin/io/element/android/features/call/impl/DefaultElementCallEntryPoint.kt

```diff
@@ -18,12 +18,15 @@ import io.element.android.libraries.di.AppScope
 import io.element.android.libraries.di.ApplicationContext
 import io.element.android.libraries.matrix.api.core.EventId
 import io.element.android.libraries.matrix.api.core.UserId
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
 import javax.inject.Inject
 
 @ContributesBinding(AppScope::class)
 class DefaultElementCallEntryPoint @Inject constructor(
     @ApplicationContext private val context: Context,
     private val activeCallManager: ActiveCallManager,
+    private val coroutineScope: CoroutineScope,
```

Comment:
Maybe stick to the `appCoroutineScope` naming?

But for this particular case, maybe we do not need a scope and `handleIncomingCall` could be `suspend`?